src/modules/organisation/persistence/organisation.repository.ts

```diff
@@ -298,10 +298,10 @@ export class OrganisationRepository {
         personPermissions: PersonPermissions,
         systemrechte: RollenSystemRecht[],
         searchOptions: OrganisationSeachOptions,
-    ): Promise<Counted<Organisation<true>>> {
```

Comment:
Please either add a named type or at least use named tuples, as a reader I have no idea what the two numbers are supposed to represent

Reply:
Using named tuple now.